src/hotspot/share/opto/superword.cpp

```diff
@@ -2418,7 +2441,6 @@ bool SuperWord::output() {
       uint vlen = p->size();
       uint vlen_in_bytes = 0;
       Node* vn = nullptr;
-      NOT_PRODUCT(if(is_trace_cmov()) {tty->print_cr("VPointer::output: %d executed first, %d executed last in pack", first->_idx, n->_idx); print_pack(p);})
```

Comment:
This was behind the wrong flag `is_trace_cmov`, and I think it was never used anyway.